src/journal-remote/log-generator.py

```diff
@@ -44,7 +40,16 @@
 bytes = 0
 counter = 0
 
-for i in range(OPTIONS.n):
+for m, (realtime_ts, (monotonic_ts, (source_realtime_ts, _))) in enumerate(
+    enumerate(
+        enumerate(
+            enumerate(range(OPTIONS.n), start=1404101101483516),
+            start=1753961140951,
+        ),
+        start=1404101101501873,
+    ),
+    start=0x198603B12D7,
+):
```

Comment:
Is this really an improvement?

Reply:
I might be more _pythonic_ than the original, but I'm not an expert :-) deferring to @keszybz who knows Python much better than I do

Reply:
maybe it would look better an less indented by leaving the variable names and declaration in place, and using then in enumerate() ? `start=m,` etc

Reply:
@bluca I don't see the point of using multiple nested `enumerate()` to make the code much harder to read. If you want to avoid the manual increment, please use a single `enumerate()` and use the starting values of the loop variables as single offsets in the one place where they are used.

Reply:
> please use a single `enumerate()` and use the starting values of the loop variables as single offsets in the one place where they are used.

@dnicolodi, please take a look at https://github.com/monosans/systemd/commit/5f40c410fef586329270ccee380d33d44c4edc3a. Is this what you mean?

Reply:
looks nicer - I'd change `m` -> `i` and avoid `start=`, use the same approach of adding the constant for all 3

Reply:
I still think this is change for the sake of change, however, this is close to what I had in mind. However, I would still use the variables to hold the initial values to serve as documentation of what these numbers are. Using `enumerate()` an an iterator returned by `range()` is also completely unnecessary as you can simply pass the starting value to `range()`. However, in this case, it is clearer to use zero as starting value for `range()` so that the offsets for the loop variables are easier to compute and understand. Namely, I would get rid of `enumerate()` all together.

Reply:
@dnicolodi, https://github.com/monosans/systemd/commit/a29c7e8f00851bbb150eebc2ad2bd03431d99818

Reply:
The `range()` argument is wrong

Reply:
> The `range()` argument is wrong

@dnicolodi, https://github.com/monosans/systemd/commit/dba1e67ea419e21bf5b0ef88f0e0cac91adfd503. Thank you for your advices, I appreciate it.